dom.go

```diff
@@ -19,9 +19,16 @@ func Render(comp Component, container *js.Object) {
 // RenderAsBody renders the given component as the body of the page, replacing
 // whatever existing content in the page body there may be.
 func RenderAsBody(comp Component) {
-	body := js.Global.Get("document").Call("createElement", "body")
+	doc := js.Global.Get("document")
+	body := doc.Call("createElement", "body")
 	Render(comp, body)
-	js.Global.Get("document").Set("body", body)
+	if doc.Get("readyState").String() == "loading" {
```

Comment:
Great, thanks for confirming that!

According to the jQuery comment, it seems it may have trouble on IE older than 9, but that's fine with me, I care about supporting only the modern evergreen browsers.